From 7a021926ad28df990fa85f013df1bbb3ddcbbf24 Mon Sep 17 00:00:00 2001
From: Robert Church <robert.church@windriver.com>
Date: Fri, 5 Apr 2019 15:26:32 -0400
Subject: [PATCH 11/11] Nova: Temporarily remove rbd_user/rbd_secret_uuid

With the update to helm v2.13.1, key deletion does not appear to be
working. 'null' values should delete the key but instead 'null' is being
merged and used as a value in the chart.

For nova, libvirt/rbd_secret_uuid and libvirt/rbd_user have no default
values and the values provided by the helm chart are not relevant to our
cluster configuration.

Temporarily need to remove these defaults from the configuration to
allow instances to boot-by-volume.

A proper ceph configuration to enable valid overrides is the next step
here as long as key deletion is busted.

NOTE: There is an associated manifest change here that is required to
remove the setting of the 'null' values for the chart.

Change-Id: Ic746bb49061633199b2f3c662840fc0a3ed2fede
Signed-off-by: Robert Church <robert.church@windriver.com>
---
 nova/values.yaml | 2 --
 1 file changed, 2 deletions(-)

diff --git a/nova/values.yaml b/nova/values.yaml
index 433ec3af..a9e86d3d 100644
--- a/nova/values.yaml
+++ b/nova/values.yaml
@@ -1512,8 +1512,6 @@ conf:
       images_type: qcow2
       images_rbd_pool: vms
       images_rbd_ceph_conf: /etc/ceph/ceph.conf
-      rbd_user: cinder
-      rbd_secret_uuid: 457eb676-33da-42ec-9a8c-9293d545c337
       disk_cachemodes: "network=writeback"
       hw_disk_discard: unmap
     upgrade_levels:
-- 
2.16.5

